lawnchair/res/drawable/ic_lens_color.xml

```diff
@@ -3,8 +3,8 @@
     xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:id="@id/qsbIconTintPrimary" android:drawable="@drawable/ic_lens_color_1" />
     <item android:id="@id/qsbIconTintQuaternary" android:drawable="@drawable/ic_lens_color_2" />
-    <item android:id="@id/qsbIconTintPrimary" android:drawable="@drawable/ic_lens_color_3" />
     <item android:id="@id/qsbIconTintTertiary" android:drawable="@drawable/ic_lens_color_4" />
+    <item android:id="@id/qsbIconTintPrimary" android:drawable="@drawable/ic_lens_color_3" />
```

Comment:
Should I attempt to fix vector asset clipping or leave it like that... it looks kinda ugly to me.

Reply:
send a screenshot of what you're talking about

Reply:
If I arrange the asset in numerical order (1 to 5)

`ic_lens_color_4` will display on top of `ic_lens_color_3`

![image](https://github.com/LawnchairLauncher/lawnchair/assets/93124920/5ce2b3cd-3118-4ad2-9f11-d11749e8c374)